Core/GDCore/Project/Project.h

```diff
@@ -65,6 +65,16 @@ class GD_CORE_API Project : public ObjectsContainer {
    */
   const gd::String& GetName() const { return name; }
 
+  /**
+   * \brief Get the categories/genres of the project.
+   */
+  const std::vector<gd::String>& GetCategories() const { return categories; };
+
+  /**
+   * \brief Get the categories of the project, to modify them (non-const).
+   */
+  std::vector<gd::String>& GetCategories() { return categories; };

```

Comment:
I don't know if it has already been used before to have this mutable array instead of a `setCategories` method

Reply:
yeah, this was done like this for other arrays too (could be slightly more performant)